### ISSUE TITLE
Does not install over website

### ISSUE THREAD

**Issue by KonradHoeffner** [orangeshirt/gnome-shell-extension-touchpad-indicator#46]
When trying to install over https://extensions.gnome.org/extension/131/touchpad-indicator/ and setting the switch to "on", application is not shown in the top bar after "Alt+F2 r" and neither is it in the extension list and neither is it found using updatedb + locate. I tried it with another extension, where it worked so it is not a browser permission problem. I use GNOME Shell 3.12.2 on Arch Linux on Dell e6530 Laptop. It also does not work when checking out from git and then Alt+F2 r.

**Comment by renatomefidf** [comment]
That's true, I just cloned the extension and I'm using it, is there a chance to update it at extensions.gnome?

Thank you!
